frontend/www/lib/ionic/scss/_menu.scss

```diff
@@ -0,0 +1,65 @@
+
+/**
+ * Menus
+ * --------------------------------------------------
+ * Side panel structure
+ */
+
+.menu {
+  position: absolute;
```

Comment:
Properties should be ordered background-color, bottom, max-height, min-height, overflow, position, top, width, z-index